hazelcast/src/main/java/com/hazelcast/multimap/impl/MultiMapService.java

```diff
@@ -338,4 +339,16 @@ public void rollbackTransaction(String transactionId) {
     public void dispatchEvent(EventData event, EntryListener listener) {
         dispatcher.dispatchEvent(event, listener);
     }
+
+    @Override
+    public Map<String, LocalMultiMapStats> getStats() {
+        Map<String, LocalMultiMapStats> mapStats = new HashMap<String, LocalMultiMapStats>();
+        for (int i = 0; i < nodeEngine.getPartitionService().getPartitionCount(); i++) {
```

Comment:
Statistics of a `MultiMap` is put in `statsMap` when a proxy is created. That's why I didn't use `statsMap`. Actually, all I need is `MultiMap` names (whether a proxy object is created for them or not). Do you know a better way to get them all?